lib/menu.rb

```diff
@@ -0,0 +1,26 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = {"starter" => 5, "mains" => 10, "desert" => 3}
+  end
+
+  def show_dishes
+    dishes.each do |dish, value|
+      puts "#{dish} : #{value}"
+    end
+  end
+
+  def check_all_dishes_are_on_menu(order_as_hash)
+    order_as_hash.each do |dish, quantity|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)